dev/com.ibm.ws.security.authentication.builtin/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -26,7 +26,10 @@ allowHashtableLoginWithIdOnly=Allow hashtable login with ID only
 allowHashtableLoginWithIdOnly.desc=Allow an application to login with just an identity in the hashtable properties. Use this option only when you have applications that require this and have other means to validate the identity.
 
 fillInGroupInformationForHashtableLogin=Automatically fill in group information for user when performing Hashtable login
-fillInGroupInformationForHashtableLogin.desc=Specifies whether the group information for the login user should be automatically populated when performing a Hashtable login. If true and no groups are provided to the Hashtable login, then the group information will be populated. If true and group information is provided, then the provided group information is used. If false, then the group information, if provided, is used. 
+fillInGroupInformationForHashtableLogin.desc=Specifies whether the group information for the login user should be automatically populated when performing a Hashtable login. If true and no groups are provided to the Hashtable login, then the group information will be populated. If true and group information is provided, then the provided group information is used. If false, then the group information, if provided, is used.
+
+useDisplayNameForSecurityName=Use displayName as securityName in customUR
+useDisplayNameForSecurityName.desc=Specifies if the displayName should be returned for programmatic APIs like getRemoteUser, getCallerPrincipal, getUserPrincipal when using Custom User Registry.
```

Comment:
To remove "should be," replace "Specifies if the displayName should be returned" with "Specifies whether to return  displayName"

Reply:
Thanks I changed it